### PR TITLE
[-] FO: Footer fix in default-bootstrap theme: nocache on copyright year

### PR DESCRIPTION
**Pull request by xGouley**
// to avoid old smarty cache displays past year, #PSCSX-6024

### CODE REVIEW

**Comment by maximebiloe** [comment]
:+1:

**Comment by tchauviere** [comment]
:+1:

**Comment by gRoussac** [comment]
Hi,

Please report on '1.6.1.x'.

Regards

File: themes/default-bootstrap/modules/blockcms/blockcms.tpl [hunk 1]
```diff
@@ -126,7 +126,7 @@
 	{if $display_poweredby}
 	<section class="bottom-footer col-xs-12">
 		<div>
-			&copy; {'Y'|date} {l s='[1]Ecommerce software by %s[/1]' mod='blockcms' sprintf=['PrestaShop™'] tags=['<a class="_blank" href="http://www.prestashop.com">']}
+			{l s='[1] %3$s %2$s - Ecommerce software by %1$s [/1]' mod='blockcms' sprintf=['PrestaShop™', 'Y'|date, '©'] tags=['<a class="_blank" href="http://www.prestashop.com">'] nocache}
```
Comment (inetbiz) [inline themes/default-bootstrap/modules/blockcms/blockcms.tpl]: I don't think special characters should be used there. &copy;?? sprintf=['PrestaShop™', 'Y'|date, '©']

**Comment by xGouley** [comment]
Reported on 1.6.1.x on new PR

File: themes/default-bootstrap/modules/blockcms/blockcms.tpl (hunk 1 above)
Comment (gRoussac) [inline themes/default-bootstrap/modules/blockcms/blockcms.tpl]: Hi,

I think it is utf-8 and so is ok ?

Regards

**Pull request closed without merging**